jdbc-postgres/src/main/resources/migrations/postgres/V19__index_execution-labels.sql

```diff
@@ -0,0 +1 @@
+create index executions_labels ON executions USING GIN((value -> 'labels'));
```

Comment:
```suggestion
create index executions_labels ON executions USING GIN(value -> 'labels');
```

Reply:
The first couple of parenthesis is for declaring the columns of the index, the second couple is the special way to define a column as a JSON Path. So both are needed.

Reply:
:+1: